docs/guided-tour/blueprints/external-blueprint/README.md

```diff
@@ -67,16 +52,46 @@ components:
+a file system representation of the component version and the second uploads this to the registry:
 
->**Tip:** If you need the same resource in multiple components, instead of copy-pasting, you might want to use 
-> [*resource configuration files*](https://ocm.software/docs/guides/getting-started-with-ocm/#using-a-resources-file).
+```
+ocm add components --create --file ../tour-ctf ../config-files/components.yaml
+ocm transfer ctf --enforce ../tour-ctf eu.gcr.io/gardener-project/landscaper/examples
+```
 
-## Referencing the Blueprint in the Installation
+We have uploaded the component version into this [location](https://console.cloud.google.com/gcr/images/gardener-project/eu/landscaper/examples/component-descriptors/github.com/gardener/landscaper-examples/guided-tour/external-blueprint) to our public OCI registry. You could download
+the component version with the following command which more or less provides you a file system representation
+of the example component version:
 
-The [Installation](./installation/installation.yaml) references the component and blueprint as follows:  
+```
+ocm download component eu.gcr.io/gardener-project/landscaper/examples//github.com/gardener/landscaper-examples/guided-tour/external-blueprint:2.0.0 -O <your location on the file system>
+```
+
+[Here](components/component) you see the downloaded component version. The perhaps most important part is the 
+[component descriptor](components/component/component-descriptor.yaml) which mainly contains the adapted data from the 
+configuration file. In its `resource` section, you find all specified artefacts including their access data. In our
+example only the reference to the blueprint is contained here.
+
+Alternatively, instead of referencing the blueprint at an external location, we can embed it in the
+component version as a so-called local blob. Therefore, in the *component configuration file* instead of an `access:...`, 
+you would have to specify an `input:...` entry, as demonstrated [here](./config-files/local-blob-components.yaml).
+The  configured path must reference the location of the blueprint directory on the file system. The creation 
+and uploading of the component version is the same as before. If you download this component version as before
+you get [this](components/local-blob-component) file system representation. Now the component descriptor references
+the blueprint as a local blob. This local blob is contained in the file system representation of the component
+version in the [blobs folder](components/local-blob-component/blobs).
+
+To embed a blueprint into a component version as a local blob is usually the preferred approach.
+
+For more details about component versions check out the [OCM Getting Started Guide](https://ocm.software/docs/guides/getting-started-with-ocm/). In the OCM documentation
+you also find more about the advantages of using components to describe your artefacts like transportability, signing, 
+scanning etc.
```

Comment:
Why are we advertising the concept of having external blueprints? Yes, this is possible, but nowhere used in a real scenario.
I would prefer that we just describe the "standard" way of embedding a blueprint ans a local blob.

I would also like to have examples embedded instead of links. The reader has to jump back and forth.
Why not give an inline example how a bloueprint has to be specified in a component version?

Reply:
It is on our list to replace all external blueprints by local ones.